contributors/devel/sig-testing/integration-tests.md

```diff
@@ -76,3 +77,13 @@ make test-integration WHAT=./test/integration/pods GOFLAGS="-v" KUBE_TEST_ARGS="
 
 If you set `KUBE_TEST_ARGS`, the test case will be run with only the `v1` API
 version and the watch cache test is skipped.
+
+### Integration testing in CI
+
+The CI job named `pull-kubernetes-integration`, which is defined in
+[https://github.com/kubernetes/test-infra/blob/master/config/jobs/kubernetes/sig-testing/integration.yaml],
```

Comment:
Is this link supposed to be embedded in some text?
Perhaps something like [`integration.yaml`](https://github.com/kubernetes/test-infra/blob/master/config/jobs/kubernetes/sig-testing/integration.yaml) (or something similar)?